--- configure.orig	2017-04-25 13:41:49.000000000 +0200
+++ configure	2017-10-28 22:29:44.417592600 +0200
@@ -2302,6 +2302,7 @@ fi
 $as_echo "$ac_cv_build" >&6; }
 case $ac_cv_build in
 *-*-*) ;;
+*_*-*) ;;
 *) as_fn_error $? "invalid value of canonical build" "$LINENO" 5;;
 esac
 build=$ac_cv_build
@@ -2335,6 +2336,7 @@ fi
 $as_echo "$ac_cv_host" >&6; }
 case $ac_cv_host in
 *-*-*) ;;
+*_*-*) ;;
 *) as_fn_error $? "invalid value of canonical host" "$LINENO" 5;;
 esac
 host=$ac_cv_host
@@ -2368,6 +2370,7 @@ fi
 $as_echo "$ac_cv_target" >&6; }
 case $ac_cv_target in
 *-*-*) ;;
+*_*-*) ;;
 *) as_fn_error $? "invalid value of canonical target" "$LINENO" 5;;
 esac
 target=$ac_cv_target
